libs/api/domains/auth-admin/src/lib/auth-admin.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common'
+import { TenantResolver } from './resolvers/tenant.resolver'
+import { TenantsService } from './services/tenants.service'
```

Comment:
I prefer grouping files by domain (tenant) rather than type (resolvers/services), but guess the monorepo is all over the place with this 🤷‍♂️🙃